pkg/patterns/declarative/pkg/manifest/patch.go

```diff
@@ -89,18 +89,22 @@ func apply(base *unstructured.Unstructured, patches []*unstructured.Unstructured
 		case err != nil:
 			return nil, err
 		default:
-			// Use Strategic-Merge-Patch to handle types w/ schema
-			// TODO: Change this to use the new Merge package.
-			// Store the name of the base object, because this name may have been munged.
-			// Apply this name to the patched object.
-			lookupPatchMeta, err := strategicpatch.NewPatchMetaFromStruct(versionedObj)
+			baseBytes, err := merged.MarshalJSON()
```

Comment:
```suggestion
			// Use Strategic-Merge-Patch to handle types w/ schema
			// Store the name of the base object, because this name may have been munged.
			// Apply this name to the patched object.
			baseBytes, err := merged.MarshalJSON()
```

I'm not sure this is better, but It may be better to remain original comments except for deleting `TODO` comment.

Reply:
Addressed